index.js

```diff
@@ -169,4 +169,32 @@ class CircuitBreaker {
     }
 }
 
-module.exports = CircuitBreaker;
+class FuseBox {
+    constructor() {
+        this.fuses = [];
+    }
+
+    createFuse(command, options) {
+        const self = this;
+        const breaker = new CircuitBreaker(command, options);
+
+        breaker.breaker.on('open', () => self.tripFuses());
```

Comment:
This is giving me a bad code smell. I feel like this is tying into something that should be a private api. CircuitBreaker should probably expose the events of its implementation directly.